pyproject.toml

```diff
@@ -1,26 +1,27 @@
 [tool.poetry]
 name = "pelican"
-version = "0.6.5"
+version = "0.6.4"
```

Comment:
Is there a reason we downgraded the version?

Reply:
Not sure why that got downgraded. Good catch! I will adjust that and ensure the PR passes!